### DISCUSSION TITLE
New DuplicateDetector extension : Feedback welcome !

### DISCUSSION THREAD

**Discussion by Poilaupat** [DevToys-app/DevToys#1305] [Show and tell]
Hi All,
First of all, thank you very much to the DevToys developers for this fantastic tool that I use almost every day since a colleague introduced me to it.
I am currently developing an extension that detects duplicates in the lines of input text, and I wanted to know if some members of the DevToys community would have some time to give me some feedback on it. All comments are welcome (features, code review, etc.). It would be really appreciated.

Nuget : [DuplicateDetectorExtension](https://www.nuget.org/packages/DuplicateDetectorExtension)
GitHub : [DuplicateDetectorExtension](https://github.com/Poilaupat/DevToys-DuplicateDetector)

The next steps are :
- Developping the CLI version of the tool
- Documentation

Thanks,

Ruben

**Comment by veler** [comment]
Hi,
I just gave it a try and it looks great so far!

**Comment by Poilaupat** [comment]
I just added a CLI version of the tool (ver 1.0.2). 
Feel free to update your installed plugin.

**Comment by ycherkes** [comment]
@Poilaupat Great extension, thank you!

Is it possible to add a "Remove Duplicates" feature?

**Comment by Poilaupat** [comment]
Hi ycherkes, glad you find the plugin useful !
The remove duplicate feature is a great idea and I think it wouldn't require too much work. 
I'm currently struggling with a broken elbow so I cannot promise you that it will be done tomorrow, but i definitely put that task on my checklist ;)

I created an issue on the project : [link](https://github.com/Poilaupat/DevToys-DuplicateDetector/issues/8#issue-2662674522)
Feel free to comment.

**Comment by Poilaupat** [comment]
# DuplicateDetector 1.0.4 Released!

Version 1.0.4 of the DuplicateDetector extension is now available!
This release introduces a "remove duplicates" feature, which is accessible in both the GUI and CLI versions of the tool.

A special thanks to @jerone for your valuable contribution to this version—your correction was greatly appreciated!

I’d also like to extend my apologies to @ycherkes for the delay in releasing this update. The end of the year was particularly busy on my side. I hope the new feature meets your expectations.

[Link to v1.0.4](https://www.nuget.org/packages/DuplicateDetectorExtension/)

**Reply by ycherkes** [reply]
@Poilaupat  @jerone 
Thank you very much - just tested the "remove duplicates" feature and it works perfectly :)
